extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1094,21 +1043,21 @@ function getDefaultSort() {
 	 * @return array: query info
 	 */
 	function getQueryInfo() {
-		$query = array(
-			"tables" => array(
-				WikiFactory::table("city_list_log"),
-				WikiFactory::table("city_list"),
-			),
-			"fields" => array( "*" ),
-			"conds" => array(
-				WikiFactory::table("city_list", "city_id" )
+		$query = [
+			"tables" => [
+				WikiFactory::table( "city_list_log" ),
+				WikiFactory::table( "city_list" ),
+			],
+			"fields" => [  "*"  ],
+			"conds" => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "conds" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)